server/elm/src/Pages/Scoreboard/View.elm

```diff
@@ -1,23 +1,536 @@
+                        [ button []
+                            [ text <| translate language Translate.NewSelection ]
+                        ]
+                    ]
+                , viewYearSelector language currentDate model.yearSelectorGap ChaneYearGap
+                , div [ class "values-percents" ] []
+                ]
+    in
+    div [ class "page-content" ]
+        [ topBar
+        , viewAggregatedChildScoreboardPane language data.entityName data.entityType
+        , viewDemographicsPane language currentDate model.yearSelectorGap data.entityType
+        , viewAcuteMalnutritionPane language currentDate model.yearSelectorGap data.entityType
+        , viewStuntingPane language currentDate model.yearSelectorGap data.entityType
+        , viewANCNewbornPane language currentDate model.yearSelectorGap data.entityType
+        , viewUniversalInterventionPane language currentDate model.yearSelectorGap data.entityType
+        , viewNutritionBehaviorPane language currentDate model.yearSelectorGap data.entityType
+        , viewTargetedInterventionsPane language currentDate model.yearSelectorGap data.entityType
+        , viewInfrastructureEnvironmentWashPane language currentDate model.yearSelectorGap data.entityType
+        ]
+
+
+viewAggregatedChildScoreboardPane :
+    Language
+    -> String
+    -> SelectedEntity
+    -> Html any
+viewAggregatedChildScoreboardPane language entityName entityType =
+    div [ class "pane" ]
+        [ viewPaneHeading language Translate.AggregatedChildScoreboard
+        , div [ class "pane-content" ]
+            [ div []
+                [ span [ class "selected-entity" ] [ text <| (translate language <| Translate.SelectedEntity entityType) ++ ":" ]
+                , span [] [ text entityName ]
+                ]
+            ]
+        ]
+
+
+formatValues : NominalDate -> Int -> List Int -> List String
+formatValues currentDate yearSelectorGap =
+    let
+        currentMonthNumber =
+            Date.monthNumber currentDate
+    in
+    List.indexedMap
+        (\index value ->
+            if yearSelectorGap == 0 then
+                if index < currentMonthNumber then
+                    String.fromInt value
+
+                else
+                    ""
+
+            else
+                String.fromInt value
+        )
+
+
+viewDemographicsPane : Language -> NominalDate -> Int -> SelectedEntity -> Html any
+viewDemographicsPane language currentDate yearSelectorGap entityType =
+    let
+        rows =
+            List.map2
+                (\item itemValues ->
+                    viewTableRow language currentDate yearSelectorGap (Translate.NCDADemographicsItemLabel item) itemValues
+                )
+                [ ChildrenUnder2, NewbornsThisMonth, LowBirthWeigh ]
+                values
+
+        values =
+            case entityType of
+                EntityVillage ->
+                    [ [ 12, 12, 14, 13, 15, 15, 15, 12, 13, 13, 14, 14 ]
```

Comment:
at first sight, it"s not obvious that what are these

Reply:
These are demo values - TIP team needed to present this feature, so they asked to have it.
Actual values are generated at follow up PRs.